client/components/gsuite/docs/new-user-list.tsx

```diff
@@ -54,7 +54,7 @@ const GSuiteNewUserListExample = () => {
 				onUsersChange={ changedUsers => setUsers( changedUsers ) }
 				users={ users }
 			>
-				{ 0 < users.length && users.every( userIsReady ) ? (
+				{ allUsersReady( users ) ? (
```

Comment:
I have the feeling we should rename that function into `areAllUsersReady()` to be consistent with `userIsReady()` but maybe that's just me. Actually, I would also rename `userIsReady()` into `isUserReady()` since we already have a `isUserComplete()` function in that same file (that's the usual naming convention for these types of functions anyway).